x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/planner/QueryTranslatorTests.java

```diff
@@ -109,16 +108,6 @@ public void testTermEqualityAnalyzer() {
         assertEquals("value", tq.value());
     }
 
-    public void testTermEqualityAnalyzerAmbiguous() {
```

Comment:
Is this tested somewhere else?

Reply:
Yes, here: https://github.com/elastic/elasticsearch/pull/39427/files#diff-334bd2f8c33b60789817181fcef4ca52R331 with 2 methods now.